docs/docs/tutorials/auth/getting_started.md

```diff
@@ -0,0 +1,157 @@
+# Setting up Custom Authentication
+
+In this tutorial, we will build a chatbot that only lets specific users access it. We'll start with the LangGraph template and add token-based security step by step. By the end, you'll have a working chatbot that checks for valid tokens before allowing access.
+
+!!! note "This is part 1 of our authentication series:"
+
+    1. Basic Authentication (you are here) - Control who can access your bot
+    2. [Resource Authorization](resource_auth.md) - Let users have private conversations
+    3. [Production Auth](add_auth_server.md) - Add real user accounts and validate using OAuth2
+
+## Setting up our project
+
+First, let's create a new chatbot using the LangGraph starter template:
+
+```bash
+pip install -U "langgraph-cli[inmem]"
+langgraph new --template=new-langgraph-project-python custom-auth
+cd custom-auth
+```
+
+The template gives us a placeholder LangGraph app. Let's try it out by installing the local dependencies and running the development server.
+```shell
+pip install -e .
+langgraph dev
+```
+> - 🚀 API: http://127.0.0.1:2024
+> - 🎨 Studio UI: https://smith.langchain.com/studio/?baseUrl=http://127.0.0.1:2024
+> - 📚 API Docs: http://127.0.0.1:2024/docs
+> 
+> This in-memory server is designed for development and testing.
+> For production use, please use LangGraph Cloud.
+
+If everything works, the server should start and open the studio in your browser.
+
+Now that we've seen the base LangGraph app, let's add authentication to it!
+
+## Adding Authentication
```

Comment:
Perhaps 1 sentence somwhere outside of the code that we're starting with hard-coded tokens for illustration purposes